queries.sql

```diff
@@ -188,4 +188,152 @@ FROM animals A
 JOIN owners O
 ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY animal DESC;
\ No newline at end of file
+ORDER BY animal DESC;
+
+/*MANY-TO-MANY QUERIES*/
+
+/*"Who was the last animal seen by William Tatcher?"*/
+
+SELECT TOP 1
+A.name AS animal_name,
+vets.name AS vet_name,
+visits.date_of_visit
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC;
+
+/*"How many different animals did Stephanie Mendez see?"*/
+
+SELECT DISTINCT
+vets.name,
+COUNT(DISTINCT A.name) AS animals_attended
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+WHERE vets.name = 'Stephanie Mendez'
+GROUP BY vets.name;
```

Comment:
- [ ] As the reviewer mentioned, the screenshot you presented does not in any way portray an understanding of the query nor does it reflect the outcome of this query. Kindly run it again and check your output. Also, kindly update the screenshot wherever possible.